deployment/install-worker.sh

```diff
@@ -1,40 +1,69 @@
 #!/bin/bash
-set +e
 
+##############################################################################
+# Description:
+# This is a helper script to install the worker binaries on your Apache Spark cluster
+#
+# Usage:
+# ./install-script.sh <cloud-provider> <path-to-worker> <local-worker-installation-path>
+#
+# Sample usage:
+# ./install-script.sh 
+#   azure 
+#   abfs://<blobcontainer>@<gen2storageaccount>.dfs.core.windows.net/<path>/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.1.0.tar.gz 
+#   /usr/local/bin
+##############################################################################
+
+set +e
 # Cloud Provider
 CLOUD_PROVIDER=$1
+
 # Path where packaged worker file (tgz) exists.
-WORKER_PATH=$2
+WORKER_PATH_OR_URI=$2
```

Comment:
I think we need SRC_ and DEST_ . LOCAL_WORKER_PATH is still not clear.

Reply:
Done